examples/examples.py

```diff
@@ -0,0 +1,86 @@
+# # ddsp-drum examples
```

Comment:
I've move the "test.py" here. It makes more sense to run it through Jupyter notebook since it's more a demo than a formal test.

(Nonetheless, it's a nice way to informally test that the code still works as you expect.)